app/code/Magento/Catalog/view/frontend/web/js/catalog-add-to-cart.js

```diff
@@ -97,7 +97,11 @@ define([
                 success: function (res) {
                     var eventData, parameters;
 
-                    $(document).trigger('ajax:addToCart', form.data().productSku, form, res);
+                    $(document).trigger('ajax:addToCart', {
+                        'sku': form.data().productSku,
+                        'form': form,
+                        'response': res
+                    });
```

Comment:
will this be taken account also in `magento/module-google-tag-manager`

- vendor/magento/module-google-tag-manager/view/frontend/web/js/google-tag-manager-cart.js 

where it expects (in M2.2.0 at least) string as 1st argument?